xcore/bump_version_pull_request/push_new_version.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# != 4 ]]; then
+    echo "usage: $0 <github actor> <github_ref> <version> <version_bumped>"
+    exit 1
+fi
+
+GITHUB_ACTOR=$1
+REF=$2
+VERSION=$3
+NEW_VERSION=$4
+git fetch --all --unshallow
+
+git checkout -b feature/bump_version_${NEW_VERSION} ${REF}
```

Comment:
I see the CI of this repo failing on case where `feature/bump_version_${NEW_VERSION}` already exists,

Would you mind adding a check if the branch already exists and you would re-create it or update it if that's the case.

Reply:
something like some indemtpotent system that never fails if it already exists ? why not.
We must check that if you have a regular repo runninggnd using this action. should we fail in this case ? thinking that the usage of this action is more important than the actual CI to test it.

Reply:
I check the code and don't want to make it indempotent here, if the branch already exists then it's a problem, someone should check it and clean it first (in the case of using this action in a regular repo).

In the CI I choosed then to just force clean the branches with the same name in case it happens.

Reply:
Sounds good to me 👍 
Most important part on my end was to ensure the CI tests do not block other PRs with unrelated work.

Reply:
make sense. thanks